cmd/tailscale/cli/file.go

```diff
@@ -286,22 +287,106 @@ func runCpTargets(ctx context.Context, args []string) error {
 	return nil
 }
 
+type onConflict string
+
+const (
+	skipOnExist         onConflict = "skip"
+	overwriteExisting              = "overwrite" //  Overwrite any existing file at the target location
+	createNumberedFiles            = "rename"    //  Create an alternately named file in the style of Chrome Downloads
+)
+
+func (v *onConflict) String() string {
+	return string(*v)
+}
+
+func (v *onConflict) Set(s string) error {
+	if s == "" {
+		*v = skipOnExist
+		return nil
+	}
+	l := onConflict(strings.ToLower(s))
+	if l != skipOnExist && l != overwriteExisting && l != createNumberedFiles {
+		return fmt.Errorf("'%s' not one of (skip|overwrite|rename)", s)
+	}
+	*v = l
+	return nil
+}
+
 var fileGetCmd = &ffcli.Command{
 	Name:       "get",
-	ShortUsage: "file get [--wait] [--verbose] <target-directory>",
+	ShortUsage: "file get [--wait] [--verbose] [--conflict=(skip|overwrite|rename)] <target-directory>",
 	ShortHelp:  "Move files out of the Tailscale file inbox",
 	Exec:       runFileGet,
 	FlagSet: (func() *flag.FlagSet {
 		fs := newFlagSet("get")
 		fs.BoolVar(&getArgs.wait, "wait", false, "wait for a file to arrive if inbox is empty")
 		fs.BoolVar(&getArgs.verbose, "verbose", false, "verbose output")
+		fs.Var(&getArgs.conflict, "conflict", "behavior when target file already exists")
 		return fs
 	})(),
 }
 
-var getArgs struct {
-	wait    bool
-	verbose bool
+var getArgs = struct {
+	wait     bool
+	verbose  bool
+	conflict onConflict
+}{conflict: skipOnExist}
```

Comment:
why do you need to initialize this? It will be done by flag.Parse before the code runs.

Reply:
This was marked as resolved, but how? Did you see this comment?